cluster/gce/gci/configure-helper.sh

```diff
@@ -2734,7 +2748,7 @@ EOF
   fi
   if [[ "${ENABLE_NODE_LOGGING:-}" == "true" ]] && \
      [[ "${LOGGING_DESTINATION:-}" == "gcp" ]]; then
-    setup-addon-manifests "addons" "fluentd-gcp"
+    setup-addon-manifests-remote "addons" "fluentd-gcp" "https://github.com/coffeepac/instrumentation-addons/tree/pc-move-fluent-gcp/fluentd-gcp"
```

Comment:
contains coffeepac in the url

Reply:
yep!  the PR to create this hasn't been merged yet.  I'm trying to make sure the `setup-addon-manifests-remote` function works in CI.  I'll swap it over to the correct fork when the other thing is mered.

Reply:
Please update.

Reply:
This still needs to be updated to the new k8s repo @coffeepac